src/dune_engine/process.ml

```diff
@@ -361,39 +416,20 @@ let pp_id id =
   let open Pp.O in
   Pp.char '[' ++ Pp.tag User_message.Style.Id (Pp.textf "%d" id) ++ Pp.char ']'
 
-module Exit_status : sig
-  type error =
-    | Failed of int
-    | Signaled of string
+module Handle_exit_status
+    (*: sig open Exit_status
```

Comment:
Did you mean to delete or uncomment this code?

Reply:
Indeed. I just commented it temporarily because merlin was insisting on colouring the whole module red while I was editing it, which was hurting my eyes. I uncommented it.